.github/workflows/push_openapi_spec_to_readme.yml

```diff
@@ -0,0 +1,36 @@
+name: Build and sync docs on PR merge or pushes to master branch
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+    types:
+      - closed
+
```

Comment:
do you actually need this? workflows with `push` run whenever a new commit lands on the specified branches -- irrespective of whether that commit came from a PR merging, or from a direct push.

Reply:
That's a good point. I didn't think about that. I'll remove the PR event trigger.